servicecontrol/license.md

```diff
@@ -1,28 +1,40 @@
 ---
-title: Install the license in ServiceControl
+title: Licensing ServiceControl
 tags:
  - servicecontrol
  - license
  - platform
 ---
 
 
-## Using the ServiceControl Management Utility
+## Licensing ServiceControl
 
-The ServiceControl Management utility allow has a simple Add License user interface  option which will import the designated license file into the registry.
+The following options outline how to add a license to ServiceControl.
+
+### ServiceControl Management Utility
+
+The ServiceControl Management utility allow has a license user interface which will import the designated license file into the registry. The license file is added to the `HKEY_LOCAL_MACHINE` registry hive so it available to be all instances of ServiceControl regardless of the service account used.
 
 ![](managementutil-addlicense.png)
 
-When the license is [installed in the registry](/nservicebus/licensing/license-management.md) it is available machine wide and applied to all Particular products running on that machine.
 
-This screen utility has a corresponding [PowerShell cmdlet](installation-powershell.md) to allow the same action to be scripted.
+### ServiceControl PowerShell
```

Comment:
Is this the same content as the installation page? Should this be a re-usable include?